migrations/Version202203021737164734_taoBackOffice.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoBackOffice\migrations;
+
+use common_report_Report as Report;
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\model\user\TaoRoles;
+use oat\tao\model\accessControl\func\AclProxy;
+use oat\tao\model\accessControl\func\AccessRule;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoBackOffice\controller\Lists;
+use oat\taoItems\model\user\TaoItemsRoles;
+
+final class Version202203021737164743_taoBackOffice extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Update global UI config';
+    }
+
+    const SHOW_OPTION = 'show';
+    const HIDE_OPTION = 'hide';
+    const GLOBAL_UI_CONFIG_NAME = 'globalUIConfig';
+
+    const DEFAULT_CONFIG = [
+        'item/image/mediaAlignment' => 'hide',
+        'test/item/BRS' => 'hide',
+        'item/multiColumn' => 'hide',
+        'item/scrollableMultiColumn' => 'hide',
+        'item/perInteractionRp' => 'hide',
+        'item/response/modalFeedbacks' => 'hide',
+        'item/commonInteractions/choiceInteraction/shufflingChoices' => 'hide',
+        'item/commonInteractions/orderInteraction/shufflingChoices' => 'hide',
+        'item/commonInteractions/extendedTextInteraction/hasMath' => 'hide',
```

Comment:
That's a great opportunity to create better names for these configs, wdyt?

Reply:
I was not able to come up with better names. Maybe you can provide an example? :)